docs/en/Configuration-Settings.md

```diff
@@ -101,153 +128,17 @@ also running at `localhost`, and enable Java remote debugging at port 7001, you
-the port number.
-
-<table class="table table-striped">
-<tr><th>Property Name</th><th>Default</th><th>Meaning</th></tr>
-{% for item in site.data.table.worker-configuration %}
-  <tr>
-    <td>{{ item.propertyName }}</td>
-    <td>{{ item.defaultValue }}</td>
-    <td>{{ site.data.table.en.worker-configuration[item.propertyName] }}</td>
-  </tr>
-{% endfor %}
-</table>
-
-
-### User Configuration
-
-The user configuration specifies values regarding file system access.
-
-<table class="table table-striped">
-<tr><th>Property Name</th><th>Default</th><th>Meaning</th></tr>
-{% for item in site.data.table.user-configuration %}
-  <tr>
-    <td>{{ item.propertyName }}</td>
-    <td>{{ item.defaultValue }}</td>
-    <td>{{ site.data.table.en.user-configuration[item.propertyName] }}</td>
-  </tr>
-{% endfor %}
-</table>
-
-### Resource Manager Configuration
-
-When running Alluxio with resource managers like Mesos and YARN, Alluxio has additional configuration options.
-
-<table class="table table-striped">
-<tr><th>Property Name</th><th>Default</th><th>Meaning</th></tr>
-{% for item in site.data.table.cluster-management %}
-  <tr>
-    <td>{{ item.propertyName }}</td>
-    <td>{{ item.defaultValue }}</td>
-    <td>{{ site.data.table.en.cluster-management[item.propertyName] }}</td>
-  </tr>
-{% endfor %}
-</table>
-
-### Security Configuration
-
-The security configuration specifies information regarding the security features, such as authentication and file permission. Settings for authentication take effect for master, worker, and user. Settings for file permission only take effect for master. See [Security](Security.html) for more information about security features.
-
-<table class="table table-striped">
-<tr><th>Property Name</th><th>Default</th><th>Meaning</th></tr>
-{% for item in site.data.table.security-configuration %}
-  <tr>
-    <td>{{ item.propertyName }}</td>
-    <td>{{ item.defaultValue }}</td>
-    <td>{{ site.data.table.en.security-configuration[item.propertyName] }}</td>
-  </tr>
-{% endfor %}
-</table>
+# Order of Configuration Sources
 
-### Configure Multihomed Networks
+If an Alluxio property is configured in multiple sources, its value gets decided by the source in
+the following order from the highest to the lowest:
 
-Alluxio configuration provides a way to take advantage of multi-homed networks. If you have more than one NICs and you want your Alluxio master to listen on all NICs, you can specify `alluxio.master.bind.host` to be `0.0.0.0`. As a result, Alluxio clients can reach the master node from connecting to any of its NIC. This is also the same case for other settings suffixed with `bind.host`.
+1. [JVM system properties (i.e., `-Dproperty=key`)](http://docs.oracle.com/javase/jndi/tutorial/beyond/env/source.html#SYS)
+2. [Environment variable](#use-environment-variables)
+3. [Property file](#use-site-property-files-recommended)
+4. [Default property value](Configuration-Propertiess.html). If no above user-specified
```

Comment:
This option doesn't belong in this list, which began with "If an Alluxio property is configured in multiple sources"